modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotHandlerContext.java

```diff
@@ -44,6 +45,9 @@ public class SnapshotHandlerContext {
     /** If {@code true} check snapshot integrity. */
     private final boolean check;
 
+    /** Request ID. */
+    private final UUID reqId;
```

Comment:
Do we actually need `requestId`? It looks like that user doesn't get this ID in log for check operations.

Reply:
Yes, we need some global operation ID. It can be compute task session id, but it has not UUID type as other snapshot operations.